### ISSUE TITLE
Audio Ducking Fails in Alphas and Betas of NVDA on 32-bit Synthesizers

### ISSUE THREAD

**Issue by Jykke67** [nvaccess/nvda#19618]
### Brief summary

• When Audio ducking mode is set to "Duck when outputting speech and sounds", nothing is ducked.
• When Audio ducking mode is set to "Always duck", all sounds including speech of NVDA are ducked.

### Steps to reproduce

1. Launch any application, such as media player, that produces non-stop sound.
2. Toggle between Audio ducking modes by pressing NVDA+Shift+d.

### Actual behavior

• When Audio ducking mode is set to "Duck when outputting speech and sounds", nothing will be ducked.
• When Audio ducking mode is set to "Always duck", all sounds, including speech of NVDA, will be ducked.

### Expected behavior

• When Audio ducking mode is set to "Duck when outputting speech and sounds", all sounds except speech and sounds of NVDA should be ducked during speech and sounds of NVDA.
• When Audio ducking mode is set to "Always duck", all sounds except speech and sounds of NVDA should always be ducked.

### NVDA logs, crash dumps and other attachments

The latest version of nvda.log attached.

### NVDA type

installed copy

### NVDA version

NVDA 2026.1beta2 (2026.1.0.54516)

### Have you tried any other versions of NVDA? If so, please report their behaviors.

Yes, the previous beta version of NVDA. The behavior is similar to the behavior of NVDA 2026.1beta2 (2026.1.0.54516).

### Windows version

Version 10.0.26200 Build 26200

### Name and version of other software in use when reproducing the issue

Firefox version 140.7.0 and others with audio/video playback running

### Other information about your system

Running Windows 11 as Administrator in normal mode

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I have restarted NVDA with add-ons disabled and the issue still occurs

### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

I have run the COM Registration Fixing Tool and the issue still occurs

**Comment by josephsl** [comment]
Hi,

Which speech synthesizer are you using? Is the synthesizer perhaps 32-bit and/or SAPI 4/5?

Thanks.

**Comment by Jykke67** [comment]
Hi,

I am using a 32-bit synthesizer, but could even use a 64-bit one. Thanks.

Have specific reasons to use a 32-bit one instead of a 64-bit one. Thanks.

**Comment by Jykke67** [comment]
Hi, and apologies for another comment,

Your question about a 32/64 bit synthesizer was more than relevant, because when I tested Audio ducking with eSpeak, it did work as expected. Thanks.

**Comment by Jykke67** [comment]
Hi again and apologies,

When testing Audio ducking modes, I did not think that this could have any connection to the bitness of the synthesizer, but it has like on Capital pitch cnange earlier that has already been fixed. Thanks.

**Comment by gerald-hartig** [comment]
This is a known and documented limitation. We plan on disabling the gesture for audio ducking for 32 bit synths in 2026.1, but fixing it properly for future releases.

**Comment by Jykke67** [comment]
Thanks. It is better if something that does not work is totally disabled than having a feature that does not work as expected.

This is perhaps something that one is able to live with until this can be fixed.
